documentation/DATA_FORMAT.md

```diff
@@ -49,17 +49,17 @@ The most important file is the **manifest**, which is a JSON file that describes
     "outliers": <relative path to outlier JSON>,    //< optional
     "centroids": <relative path to centroids JSON>, //< optional
     "bounds": <relative path to bounds JSON>        //< optional
+    "backdrops": <array of backdrop image sets>     //< optional, see 2. Backdrops for more details
 }
-
-
-*Note: all paths are relative to the location of the manifest file.
 ```
 
+*Note: all paths are relative to the location of the manifest file.*
```

Comment:
is this strictly true or are they allowed to be absolute urls also?

Reply:
Currently this is strictly true, there's no support for absolute URLs.